_sass/_article.scss

```diff
@@ -22,4 +22,18 @@
 			color: #0c7abf;
 		}
 	}
+	&_contact {
+		margin: 0;
+		list-style-type: none;
+		li{
```

Comment:
Can we either have `decleration{` or `decleration {` but keep consistent with one.